crates/utils/src/lib.rs

```diff
@@ -1,5 +1,12 @@
+#![cfg_attr(docsrs, feature(doc_cfg))]
+
 pub mod errors;
 pub mod iter;
+pub mod format {
+    mod json;
```

Comment:
The fact that this uses JSON feels like an implementation detail to me; I think `serde` would be a better name.

Reply:
I agree that serde is a better. But it's not exposed, so implementation detail is only visible if inspecting the source code, as it should be, so I think it's fine

Reply:
Oops, sorry, didn't notice that the `mod json` was private.